recipes/_configure_limits.rb

```diff
@@ -0,0 +1,11 @@
+# coding: UTF-8
+#
+# Cookbook Name:: cerner_splunk
+# Recipe:: _configure_limits
+#
+# Configures the limits.conf settings for the system
+
+splunk_template 'system/limits.conf' do
+  stanzas node['splunk']['config']['limits']
```

Comment:
1) We probably need to default this attribute so that it's not required to be set to run.
2) If empty, should we remove the limits file? 
3) Does this belong under config or just under splunk (with apps and monitors)